templates/common/.devcontainer/Dockerfile/base/Dockerfile

```diff
@@ -1,4 +1,6 @@
 ARG VARIANT=bullseye
```

Comment:
Will prob need in /Dockerfile/func/Dockerfile too.

Reply:
Yup - good catch!